doc/api/errors.md

```diff
@@ -193,7 +193,7 @@ detailing the point in the code at which the `Error` was instantiated, and may
 provide a text description of the error.
```

Comment:
This commit has the latest review comment fixes. 
@bnoordhuis, i believe i've addressed all of you comment. Thanks for the review.